src/config/dynamic-theme-fixes.config

```diff
@@ -15197,6 +15197,13 @@ IGNORE INLINE STYLE
 
 ================================
 
+tabletochki.org
+
+INVERT
+.perekaz__list img
```

Comment:
Where can I find this element?

Reply:
The CSS selects all the logos in the "Other ways to make a donation" section on https://tabletochki.org/en/donate_now/

Reply:
Just FYI, you may already know this, but in case you don't... Firefox's integrated devtools allow you to type in any CSS selector and it will find all occurrences in the DOM.  It even works with complex selectors.  Even better, it has auto-suggest so you don't have to type the entire selector in most cases.

Reply:
I did knew, but I wasn't aware on which page this element was on.

Reply:
Ah, thanks.  I put the page's URL on the issue report, but not on the pull request.  Next time, I'll be sure to include it in the pull request's comment section too.